lib/checks/keyboard/page-no-duplicate.js

```diff
@@ -4,7 +4,16 @@ if (!options || !options.selector || typeof options.selector !== 'string') {
 	);
 }
 
-let elms = axe.utils.querySelectorAll(virtualNode, options.selector);
+// only look at the first node and it's related nodes
+const key = 'page-no-duplicate;' + options.selector;
+if (axe._cache.get(key)) {
+	return 'ignored';
```

Comment:
Hmm, not too keen on having strings as the return value of checks. I think we should use date() for this.